packages/packagers/js/src/prelude.js

```diff
@@ -24,6 +24,8 @@
   var previousRequire =
     typeof globalObject[parcelRequireName] === 'function' &&
     globalObject[parcelRequireName];
+
+  cache = previousRequire.cache || {};
```

Comment:
can we avoid passing the cache in as an argument to the main prelude iife and just initialize it here in the case there's no previous require?

cc @devongovett